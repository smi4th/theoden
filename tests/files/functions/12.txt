fonction a(arg) {
    if (arg == 1) {
        return 1;
    } else {
        return a(arg - 1) + arg;
    }
}
v = a(10);