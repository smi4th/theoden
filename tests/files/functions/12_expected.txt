---TEST OUTPUT---
---MEMORY---
v = ('int', 1)
---FUNCTIONS---
a = (('param', 'arg'), ('linst', ('if', ('==', ('var', 'arg'), ('int', 1)), ('linst', ('return', ('int', 1))), 'empty'), ('linst', ('return', ('int', 0)))))
